include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -266,6 +266,13 @@ def TTIR_SigmoidOp: TTIR_ElementwiseUnaryOp<"sigmoid"> {
     }];
 }
 
+def TTIR_ExponentialOp: TTIR_ElementwiseUnaryOp<"exponential"> {
```

Comment:
Please rename `ExponentialOp` into `ExpOp` across the PR. We want to maintain parity with ttnn lib (which tracks torch).